src/main/java/tech/gruppone/stalker/server/model/Connection.java

```diff
@@ -0,0 +1,32 @@
+package tech.gruppone.stalker.server.model;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.NonNull;
+
+//TODO verify how I can make a union with the model ConnectedOrganization (the same for the interface repository and the controller!)
+public class Connection {
+
+  @Id
+  @NonNull
+  private Long organizationId;
+  @NonNull
+  private Long userId;
+
```

Comment:
manca il campo creationDate. come lo gestiamo? in teoria viene generato automaticamente dall'INSERT

Reply:
E' generato automaticamente dall'insert. Non serve.

Reply:
questa classe è usata sia nelle select che nelle insert. nelle insert non serve, ma nelle select si